doc/source/conf.py

```diff
@@ -294,6 +296,17 @@
 autodoc_typehints = "signature"
 
 
+def filter_out_undoc_class_members(member_name, class_name, module_name):
+    module = import_module(module_name)
```

Comment:
Question: Any danger of import errors (external library integration that isn't installed)? Will sphinx build raise the error?

Reply:
Good question, I'm not sure. I think we will see when it happens.